### DISCUSSION TITLE
Using hcaptcha in a form made with Livewire, it happens that when a field loses focus the hcaptcha div disappears

### DISCUSSION THREAD

**Discussion by faridsa** [livewire/livewire#5704] [1. Help]
As said in the title, that is the problem.

I've tried using `wire:model.defer` and that works on most cases, unless the form has some file input field. In that case the problem reappears.

The same occurs when binding hcaptcha response to a Livewire variable, when that variable changes, hcaptcha div disappears

Another strange case (at least for me) is what happen in a multi step form, I need hcaptcha in the last step, but it not renders

Any idea on how to manage it?

**Comment by joshhanley** [comment]
@faridsa try using `wire:ignore` on your hcaptcha div

**Reply by faridsa** [reply]
Thank you for your input @joshhanley! It works fine in all normal forms (using firefox and chrome) but not in multistep forms. Anyway is a big advance!
